lib/lab_manager/app/endpoints/uptime.rb

```diff
@@ -19,6 +19,10 @@ class Uptime < Base
             [500, "Error: #{err.message}"]
           end
         end
+
+        get '/computes' do
+          halt 200, ::Compute.where(state: params[:status]).map(&:id).to_json
```

Comment:
Really map(&:id)? I would expect to get the whole resource.

What happens when params[:status] is undefined?